adapters/repos/db/clusterintegrationtest/backup_coordinator_integration_test.go

```diff
@@ -146,7 +146,7 @@ func TestDistributedBackups(t *testing.T) {
 					if time.Now().After(start.Add(30 * time.Second)) {
 						t.Fatal("backup deadline exceeded")
 					}
-					resp, err := node.scheduler.BackupStatus(ctx, &models.Principal{}, "fake-backend", backupID)
+					resp, err := node.scheduler.BackupStatus(ctx, &models.Principal{}, "fake-backend", backupID, "", "") // FIXME
```

Comment:
can we remove the `FIXME` comment?

Reply:
Should I add some tests here?  I test at the module level, not at coordinator

Reply:
if you have time you can add tests at a coordinator level, but imo it's not mandatory